openedx/adg/lms/courseware_override/views.py

```diff
@@ -0,0 +1,77 @@
+"""
```

Comment:
The application was courseware_override, so keeping the context in mind, it made more sense to name it views. Because, technically we're overriding the views file of courseware app.

Reply:
I guess we should discuss this within the team.